### PR TITLE
Mib 593

### PR DESCRIPTION
**Pull request by PasGat**

### CODE REVIEW

File: app/uk/gov/hmrc/merchandiseinbaggage/service/CalculationService.scala [hunk 1]
```diff
@@ -34,6 +36,11 @@ class CalculationService @Inject()(connector: CurrencyConversionConnector)(impli
     calculationRequest.goods.purchaseDetails.currency.valueForConversion
       .fold(Future(calculation(calculationRequest, BigDecimal(1), None)))(code => findRateAndCalculate(calculationRequest, code, date))
 
+  //TODO to be enhanced to handle exports too!
+  def calculateThreshold(calculationResults: Seq[CalculationResult], destination: Option[GoodsDestination]): ThresholdCheck =
+    if (calculationResults.map(_.gbpAmount.value).sum > destination.getOrElse(GreatBritain).threshold.value) OverThreshold
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/merchandiseinbaggage/service/CalculationService.scala:41]: can we replace `calculationResults.map(_.gbpAmount.value).sum` with `totalGbpValue` ?

Comment (sureshhmrc) [reply app/uk/gov/hmrc/merchandiseinbaggage/service/CalculationService.scala:41]: why are we not handling exports in this PR ?

Comment (PasGat) [reply app/uk/gov/hmrc/merchandiseinbaggage/service/CalculationService.scala:41]: There is a TODO - I tried and was huge change to the model so reverted and decided to do it more gradually - at the moment BE only knows about importGoods

**Comment by sureshhmrc** [comment]
that's fine, looks like a test failure, 
have you fixed it already ?

**Review by sureshhmrc — APPROVED** [review]

**Pull request merged**